src/main/java/com/vmtapp/enterprise/TicketController.java

```diff
@@ -35,9 +35,9 @@ public String saveTicket(Ticket ticket){
         try{
             ticketService.save(ticket);
         }catch(Exception e){
-
+            e.printStackTrace();
```

Comment:
It's important to capture error information in catch blocks.  Take this a few steps further:
Use proper logging instead of printStackTrace().
- Loggins has the concept of priority (error, debug, warn, etc.), which printStackTrace doesn't have.
- No one will see printStackTrace(), as it goes to console. 
- Log messages can be aggregated by several commercial tools.  Not so easy with printStackTrace.